src/pages/home/platform-carousel/_utils.tsx

```diff
@@ -145,10 +171,20 @@ export const getPlatformDetails = (no_of_copies) => {
     return new_details
 }
 
-export const no_slide_sets = 11
+export const no_slide_sets = () => {
+    const { is_row } = getCountryRule()
+    if (!is_row) {
+        return 1
+    }
+    return 11
+}
 
 export const getSlideStartingIndex = () => {
-    return Math.round((no_slide_sets * 8) / 2 - 2)
+    const { is_row } = getCountryRule()
+    if (!is_row) {
+        return 1
+    }
+    return Math.round((no_slide_sets() * 8) / 2 - 2)
```

Comment:
Can this be coded dynamically based on the number of current slides?

Reply:
Since we have only 8 slides, they've made this Math.round to make it 88 to simulate a loop.
We need both dynamically based number of slides as you have said and also some refactoring on the carousel to make it a real loop instead of 88 slides!
Can we make another card to refactor the carousel?